config/locales/en.yml

```diff
@@ -68,6 +68,18 @@ en:
       home:
         statistics:
           plans_count: Plans
+    proposals:
+      proposals:
+        orders:
+          label: 'Order proposals by:'
+          most_commented: Most commented
+          most_endorsed: Most endorsed
+          most_equitable: Most equitable
```

Comment:
ℹ️ Same here. I tried to add most equitable by itself and it lost all the other translations at this level. Looks like if you're going to add a key you have to add the whole local hash.